mozetl/clientsdaily/rollup.py

```diff
@@ -0,0 +1,121 @@
+import datetime as DT
+from pyspark.sql import SparkSession
+from pyspark.sql import functions as F
+import click
+from moztelemetry.standards import filter_date_range
+from mozetl.utils import generate_filter_parameters, format_spark_path
+from fields import MAIN_SUMMARY_FIELD_AGGREGATORS
+
+ACTIVITY_SUBMISSION_LAG = DT.timedelta(10)
+MAIN_SUMMARY_VERSION = 4
+MAIN_SUMMARY_PATH = "s3://telemetry-parquet/main_summary/v{}".format(
+    MAIN_SUMMARY_VERSION)
+WRITE_VERSION = '2'
+STORAGE_BUCKET = 'net-mozaws-prod-us-west-2-pipeline-analysis'
+STORAGE_SUB_DIR = '/spenrose/clients-daily/v{}/'.format(WRITE_VERSION)
```

Comment:
I've seen the term `prefix` used more widely than `sub_dir` for s3 operations (`boto` for example).